packages/storybook-helpers/decorators/ThemeDecorator.tsx

```diff
@@ -33,15 +33,13 @@ type ThemeSetterProps = {
   theme: Theme;
 };
 const ThemeSetter = ({ theme }: ThemeSetterProps) => {
-  const { setTheme } = useTheme();
```

Comment:
Good 👁️, thanks! As discussed, any of the stories with the `ThemeSwitcher` component were actually previously incorrect due to the  infinite re-render bug in `ThemeDecorator` where the initial state should have been the system default rather than light mode. The home page I've added a Chromatic delay to the `DateInput` and `Pending` stories that should get them to where they should be (more or less). The other diffs are existing issues: https://github.com/GCTC-NTGC/gc-digital-talent/issues/9636, https://github.com/GCTC-NTGC/gc-digital-talent/issues/10025.